src/common/colorspaces.c

```diff
@@ -38,6 +38,7 @@
 #endif
 
 static const cmsCIEXYZ d65 = {0.95045471, 1.00000000, 1.08905029};
+//static const cmsCIEXYZ d50 = {0.96422000, 1.00000000, 0.82490000};
```

Comment:
I tried to use the same structure of code that is used for Adobe RGB, but actually none of these constants are actually used in the code, which is strange because we are supposed to correct the XYZ illuminant (D50 or D65) at some point when applying the conversion matrices between color spaces. So anyway, I dropped that here only for consistency, but they might come in handy at some point.